pepper/apps/hash_transform.h

```diff
@@ -1,11 +1,16 @@
 #include <stdint.h>
 
 #ifndef TEST
-#define ORDERS 725
+#define ORDERS 2
 #endif
 
+#ifdef TEST
+typedef long int128;
```

Comment:
ditto.

Reply:
I think you might have to import common.h here for it to pick up the uint typedef from there